libs/go/ztsroletoken/Makefile

```diff
@@ -0,0 +1,84 @@
+#
+# Makefile to build ZTS Role Token client utility
+# Prerequisite: Go development environment
+#
+# Copyright 2016 Yahoo Inc.
```

Comment:
should be 2017 Yahoo Holdings, Inc

please make the change in all other files as well.